src/irmin-pack/unix/stats_intf.ml

```diff
@@ -93,25 +193,55 @@ module type Sigs = sig
   end
 
   module Index : sig
-    include module type of Index
+    include module type of Index with type t = Index.t
 
     type stat
 
     val export : stat -> t
   end
 
   module File_manager : sig
-    include module type of File_manager
+    include module type of File_manager with type t = File_manager.t
 
     type stat
 
     val export : stat -> t
   end
 
+  module Latest_gc : sig
+    include
+      module type of Latest_gc
+        with type duration = Latest_gc.duration
+        with type ocaml_gc = Latest_gc.ocaml_gc
+        with type rusage = Latest_gc.rusage
+        with type step = Latest_gc.step
+        with type worker = Latest_gc.worker
+        with type stats = Latest_gc.stats
+        with type t = Latest_gc.t
+
+    type stat
+
+    val export : stat -> t
+    val new_suffix_end_offset_before_finalise : worker -> int63
+    val finalise_duration : stats -> float
```

Comment:
Can you add a doc to this? The most important being the unit of the time.

Reply:
Done, and fixuped!